config/config/sc-config.yaml

```diff
@@ -1254,14 +1258,20 @@ networkPolicies:
   rcloneSync:
     enabled: true
     destinationObjectStorageS3:
-      ips: [] # - "set-me-if-objectStorage.sync.enabled-and-type-is-s3"
-      ports: [] # - 443
+      ips:
+        - "set-me-if-objectStorage.sync.enabled-and-type-is-s3"
+      ports:
+        - 443
```

Comment:
They get templated into networkpolicies but only for the types rclone-sync uses(so if only s3 gets used it will only template the s3 part) and only if those types doesn't have their corresponding keys in the nondefault sc-config.yaml. From what i can see by testing this pr.

Reply:
Yes, the if clauses should prevent this from being templated when sync is not enabled